extension/e2e-tests/sendPayment.test.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from "./test-fixtures";
+import { login, PASSWORD } from "./helpers/login";
+
+test("Send XLM payment", async ({ page, extensionId }) => {
+  await login({ page, extensionId });
```

Comment:
Unfortunately, I also couldn't run this `login` function in a `before` hook. The background state does not seem to carry over. When I would run the test, it would just take me right back to onboarding.

Maybe the solution here is to run one big test that logs in, sends a payment, does a swap, and then checks account history in one swoop? Again, not an amazing solution - but at the very least one way to give us a little coverage while we figure out how to make these tests better

Reply:
Hmm interesting, yeah I mean on the one hand I think not relying on mocks makes for a better test but yeah it is definitely more involved. I find it weird that the background scope can't be mocked or retained though. Is this on manifest v2? I might expect this on v3 since it's service workers and they go away periodically but find it weird for v2.